client/OPUNetTransportLayer.h

```diff
@@ -44,7 +44,7 @@ class OPUNetTransportLayer : public NetTransportLayer
 	// Following functions: Return true on success, false on failure
 	bool CreateSocket();
 	bool HostGame(USHORT port, const char* hostPassword, const char* creatorName, int maxPlayers, int gameType);
-	bool SearchForGames(char* hostAddress, USHORT defaultHostPort);
+	bool SearchForGames(char* hostAddressString, unsigned short defaultHostPort);
```

Comment:
I noticed `HostGame` also uses `USHORT`. Seems odd having two functions right next to each other using different type names for what should be the same type.

Reply:
Yeah, I thought about matching them in this branch but decided to open the issue for the final solution and just get them matching in this branch.